lisk/package.json

```diff
@@ -28,9 +28,11 @@
 	"scripts": {
 		"start": "node src/index.js",
```

Comment:
this start also need to be changed, and related build, docker, pm2 setting need to be changed accordingly